themes/classic/templates/_partials/microdata/product-jsonld.tpl

```diff
@@ -53,8 +53,8 @@
     {if $hasAggregateRating},
     "aggregateRating": {
       "@type": "AggregateRating",
-      "ratingValue": "{$ratings.avg|round:1|escape:'html':'UTF-8'}",
-      "reviewCount": "{$nbComments|escape:'html':'UTF-8'}"
+      "ratingValue": "{$product.productComments.averageRating|round:1|escape:'html':'UTF-8'}",
+      "reviewCount": "{$product.productComments.nbComments|escape:'html':'UTF-8'}"
```

Comment:
To make it work with the module < 5, verify if these keys exist, if they not, use past keys